mne/viz/ica.py

```diff
@@ -322,6 +322,11 @@ def plot_ica_properties(ica, inst, picks=None, axes=None, dB=True,
         The ICA solution.
     inst : instance of Epochs or Raw
         The data to use in plotting properties.
+
+        .. note::
+           You can interactively cycle through topographic maps for different
+           channel types by pressing :kbd:`T`.
+
```

Comment:
```suggestion
```
I'm surprised that numpydoc didn't complain about this... maybe it's from the `[skip actions]`. In any case I'll push and let all CIs run